osu.Game/Graphics/UserInterface/BackButton.cs

```diff
@@ -20,7 +20,7 @@ public BackButton(Receptor receptor = null)
         {
             Size = TwoLayerButton.SIZE_EXTENDED;
 
-            Child = button = new TwoLayerButton
+            Child = button = new TwoLayerButton(HoverSampleSet.Submit)
```

Comment:
Only question is this one: is this for future safety? Currently the only usage of `TwoLayerButton` is this back button.